waspc/test/AppSpec/ValidTest.hs

```diff
@@ -188,13 +188,16 @@ spec_AppSpecValid = do
     basicAppSpec =
       AS.AppSpec
         { AS.decls = [basicAppDecl],
-          AS.externalCodeDirPath = systemSPRoot SP.</> [SP.reldir|test/src|],
-          AS.externalCodeFiles = [],
+          AS.waspProjectDir = fromJust $ parseAbsDir "/wasp-project",
```

Comment:
@Martinsos and @shayneczyzewski, this missing was causing a warning when running tests. Is it ok to set it to a dummy value like this? 

NOTE: I do the same thing in `WebGeneratorTest.hs`.

Reply:
Hey thanks, it was something I did not catch that Martin just told me about. If you want to fix it in yours you can save me this merge. Either way works. Here is how I did it: https://github.com/wasp-lang/wasp/pull/801/
Thanks!

Reply:
Merged and resolved